tests/tracking/test_fluent.py

```diff
@@ -58,14 +73,57 @@ def test_get_experiment_id_from_env():
         assert _get_experiment_id_from_env() == exp_id
 
 
-def test_get_experiment_id():
-    # When no experiment is active should return default
-    assert _get_experiment_id() == Experiment.DEFAULT_EXPERIMENT_ID
-
+def test_get_experiment_id_with_active_experiment_returns_active_experiment_id():
     # Create a new experiment and set that as active experiment
     with TempDir(chdr=True):
         name = "Random experiment %d" % random.randint(1, 1e6)
         exp_id = mlflow.create_experiment(name)
         assert exp_id is not None
         mlflow.set_experiment(name)
         assert _get_experiment_id() == exp_id
+
+
+def test_get_experiment_id_with_no_active_experiments_returns_default_experiment_id():
+    assert _get_experiment_id() == Experiment.DEFAULT_EXPERIMENT_ID
+
+
+def test_get_experiment_id_in_databricks_detects_notebook_id_by_default():
```

Comment:
Btw, these tests look quite similar to #913. May not need both sets for the DB integration, or may want to find the overlaps.

Reply:
Actually, I guess it doesn't quite overlap, as this tests specifically for the autodetection/overriding behavior.